<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Login"
           tests="3"
           failures="1"
           errors="0"
           skipped="2"
           time="0.000">
  <properties />
            <testcase name="init" time="0.000" classname="com.neso.testcase.LoginCase">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.neso.factory.CommonUtil.initBrowser(Unknown Source)
  com.neso.testcase.LoginCase.init(Unknown Source)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:298)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.testng.TestNG.privateMain(TestNG.java:1355)
  org.testng.TestNG.main(TestNG.java:1324)
]]>
      </failure>

          </testcase>
  
            <testcase name="checkLogin" time="0.0" classname="com.neso.testcase.LoginCase">
          <skipped />
    </testcase>
            <testcase name="close" time="0.0" classname="com.neso.testcase.LoginCase">
          <skipped />
    </testcase>
  
  
</testsuite>
